### PR TITLE
updating loggedOutUserCanCreateWiki automated test case

### PR DESCRIPTION
**Pull request by justnpT**
Please don't merge yet

- Removing validation test cases
CreateNewWiki_002_wrongPassword
CreateNewWiki_003_blankPassword
CreateNewWiki_004_blankUserName
CreateNewWiki_005_invalidUserName
Ticket to recover them for new auth modal:
https://wikia-inc.atlassian.net/browse/SOC-2322

anonCanRemindPasswordWhileCreatingWiki
- removing this test case because reminding password is checked by a separate test
 
anonCanSignUpWhenCreatingNewWiki
- removing this test case because it can not be updated until the infrastructure is prepared for registration through auth modal:
https://wikia-inc.atlassian.net/browse/SOC-2325

verification:
http://jenkins:8080/view/Social/job/Test-Case-Automation-Login-Signup/22/

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java [hunk 1]
```diff
@@ -98,34 +98,34 @@ public void anonCanSignUp() {
...
-    createNewWiki1 = confirmPageAlmostThere.CNWSubmitButton(email, emailPassword);
-    createNewWiki1.verifyWikiName(wikiName);
+//    CreateNewWikiLogInSignUpPageObject cnwSignUpPage = createNewWiki1.clickNextToSignIn();
+//    SignUpPageObject signUp = cnwSignUpPage.submitSignup();
+//    String userName = "User" + signUp.getTimeStamp();
+//    String password = "Pass" + signUp.getTimeStamp();
+//    String email = credentials.emailQaart2;
+//    String emailPassword = credentials.emailPasswordQaart2;
+//
+//    signUp.typeEmail(email);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java [hunk 2]
```diff
@@ -98,34 +98,34 @@ public void anonCanSignUp() {
...
+//    signUp.typeUserName(userName);
+//    signUp.typePassword(password);
+//    signUp.enterBirthDate(PageContent.WIKI_SIGN_UP_BIRTHMONTH, PageContent.WIKI_SIGN_UP_BIRTHDAY,
+//        PageContent.WIKI_SIGN_UP_BIRTHYEAR);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java [hunk 3]
```diff
@@ -98,34 +98,34 @@ public void anonCanSignUp() {
...
+//    AlmostTherePageObject almostTherePage = signUp.submit(email, emailPassword);
+//    almostTherePage.verifyAlmostTherePage();
+//    ConfirmationPageObject confirmPageAlmostThere =
+//        almostTherePage.enterActivationLink(email, emailPassword, wikiCorporateURL);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java [hunk 4]
```diff
@@ -98,34 +98,34 @@ public void anonCanSignUp() {
...
-        PageContent.WIKI_SIGN_UP_BIRTHYEAR);
-    AlmostTherePageObject almostTherePage = signUp.submit(email, emailPassword);
-    almostTherePage.verifyAlmostTherePage();
-    ConfirmationPageObject confirmPageAlmostThere =
-        almostTherePage.enterActivationLink(email, emailPassword, wikiCorporateURL);
-    confirmPageAlmostThere.typeInUserName(userName);
-    confirmPageAlmostThere.typeInPassword(password);
-    createNewWiki1 = confirmPageAlmostThere.CNWSubmitButton(email, emailPassword);
-    createNewWiki1.verifyWikiName(wikiName);
+//    CreateNewWikiLogInSignUpPageObject cnwSignUpPage = createNewWiki1.clickNextToSignIn();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java [hunk 5]
```diff
@@ -5,91 +5,94 @@
...
+//    cnwLogin.verifyBlankPasswordValidation();
+    }
+
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_004"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_004_blankUserName() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+//    cnw1.verifySuccessIcon();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java [hunk 6]
```diff
@@ -5,91 +5,94 @@
...
-    cnwLogin.verifyInvalidUserNameValidation();
-  }
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_003"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_003_blankPassword() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+        cnw1.verifySuccessIcon();
+//    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.clickNextToSignIn();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java [hunk 7]
```diff
@@ -5,91 +5,94 @@
...
-    cnwLogin.verifyEmptyUserNameValidation();
-  }
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_002"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_002_wrongPassword() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+        cnw1.verifySuccessIcon();
+//    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.clickNextToSignIn();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java [hunk 8]
```diff
@@ -5,91 +5,94 @@
...
+    }
+
+    @Test(groups = {"CNW", "CreateNewWikiLoggedOut_005"}, enabled = false)
+    @RelatedIssue(issueID = "SOC-2283", comment = "Test manually: Test is failing do to out of date test")
+    public void CreateNewWiki_005_invalidUserName() {
+        WikiBasePageObject base = new WikiBasePageObject();
+        CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
+        cnw1.typeInWikiName(cnw1.getWikiName());
+        cnw1.verifySuccessIcon();
+//    CreateNewWikiLogInSignUpPageObject cnwLogin = cnw1.clickNextToSignIn();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java [hunk 9]
```diff
@@ -106,10 +107,16 @@ public CreateNewWikiPageObjectStep2 submit() {
     return new CreateNewWikiPageObjectStep2(driver);
   }
 
-  public CreateNewWikiLogInSignUpPageObject submitToLogInSignUp() {
+//  public CreateNewWikiLogInSignUpPageObject clickNextToSignIn() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java [hunk 10]
```diff
@@ -1,6 +1,7 @@
 package com.wikia.webdriver.testcases.logintests;
 
 import com.wikia.webdriver.common.contentpatterns.CreateWikiMessages;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.contentpatterns.CreateWikiMessages'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java [hunk 11]
```diff
@@ -1,6 +1,7 @@
...
+import com.wikia.webdriver.common.core.Assertion;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.Assertion'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 16 issues:
* MAJOR 10 major
* MINOR 6 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep2'.](https://github.com/Wikia/selenium-tests/blob/f77ac8ae1bbd4b1d2811ff4e6a44c88ed75bd8c4/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L18) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject'.](https://github.com/Wikia/selenium-tests/blob/f77ac8ae1bbd4b1d2811ff4e6a44c88ed75bd8c4/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L15) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep3'.](https://github.com/Wikia/selenium-tests/blob/f77ac8ae1bbd4b1d2811ff4e6a44c88ed75bd8c4/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L19) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/f77ac8ae1bbd4b1d2811ff4e6a44c88ed75bd8c4/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L16) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MAJOR [2 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/f77ac8ae1bbd4b1d2811ff4e6a44c88ed75bd8c4/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java) [rule](http://sonar-s1/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

File: src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java [hunk 12]
```diff
@@ -5,91 +5,43 @@
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.AuthModal;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/1216d2c27de311748cbd34028a96dd2fef3bdade/src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java#L4) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/409bca27d50f23103cdf7a27fd6be85219705ad5/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L17) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/1413763b2f31bd88828ee3b78d21e48dfc271c76/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/1413763b2f31bd88828ee3b78d21e48dfc271c76/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L12) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by bkoval** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java (hunk 12 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java (hunk 10 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.contentpatterns.CreateWikiMessages'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java (hunk 11 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.core.Assertion'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 11 issues:
* MINOR 11 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java#L4) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L17) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L16) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L15) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep2'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L18) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep3'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L19) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L12) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/ff714b0646826877887418442ca74a95992c0cfd/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by bkoval** [comment]
LGTM, but please resolve problems found by SQ

**Comment by WikiaTech** [comment]
SonarQube analysis reported 11 issues:
* MINOR 11 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java#L4) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L15) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep3'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L19) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep2'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L18) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L16) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L17) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/eda2c797c094d8fd177b2fb7cdf63bc508e6eea2/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L12) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by kvas-damian** [comment]
LGTM, but please fix errors reported by Sonar.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 11 issues:
* MINOR 11 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_loggedOutUser.java#L4) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L16) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep2'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L18) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L15) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep3'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L19) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java#L17) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L12) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/4d15aa4e1626c015deb73fb253c9e6434784ef07/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1'.](https://github.com/Wikia/selenium-tests/blob/55686d8e38cd6c34d4c87abdd547d045409d6b90/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L12) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiLogInSignUpPageObject'.](https://github.com/Wikia/selenium-tests/blob/55686d8e38cd6c34d4c87abdd547d045409d6b90/src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by justnpT** [comment]
re verified:
http://jenkins:8080/view/Social/job/Test-Case-Automation-Login-Signup/30/

**Comment by justnpT** [comment]
The test from group Signup_anonCanSignUp is failing because the environment became more unstable, but this issue is not a part of the SOC-2283. Merging.

**Comment by kvas-damian** [comment]
:dancer:

**Pull request merged**